### ISSUE TITLE
Use `rustfmt` when translating

### ISSUE THREAD

**Issue by LegNeato** [immunant/c2rust#742]
`rustfmt` used to be part of the refactoring tool but currently isn't part of refactoring _or_ translation. I believe that formatting with `rustfmt` should be part of the translation tool. Doing so will make reading the generated code nicer, makes it so the transpiler code doesn't ever need to care about formatting, and matches user expectations.

**Comment by kkysen** [comment]
I agree.  Given the refactoring engine is not currently supported, and running `rustfmt` on the transpiler output is quite easy, I think this is something we should do.  That said, it is quite easy to do yourself, too.
